status/status.go

```diff
@@ -216,3 +220,18 @@ func GenerateSummary(state intgteststat.IntegrationTestStatus, snapshotName, sce
 
 	return summary, nil
 }
+
+// GenerateURLrefferenceToConsole returns string link to integration test pipelinerun logs for tenant
+func GenerateURLrefferenceToConsole(application string, pipelinerun string, namespace string) string {
+	console_url := os.Getenv("CONSOLE_URL")
+	if console_url == "" {
+		return "ENVVARNOTFOUND"
+	}
+
+	papw := "/preview/application-pipeline/workspaces/"
```

Comment:
What happened to idea, that we will use template to construct URL instead of hardcoding it in code?